### PR TITLE
fix(services.aapi): Aapi services get method return an array

### PR DESCRIPTION
**Pull request by cbourgois**
### Requirements

* [ ] Need `/services` 2api to be released

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,9 +32,10 @@
     "grunt-ng-annotate": "~0.10.0",
     "grunt-wiredep": "^2.0.0",
     "jasmine-core": "^2.6.4",
-    "matchdep": "^0.3.0"
+    "matchdep": "^0.3.0",
+    "grunt-injector": "^0.6.0"
   },
   "dependencies": {
-    "grunt-injector": "^0.6.0"
+    "lodash": "^3.10.1"
```
Comment (FredericEspiau) [inline package.json]: ?

Comment (antleblanc) [reply package.json]: `ovh-api-services` is using `lodash` as dependency. Good catch @cbourgois 👍.

**Comment by antleblanc** [comment]
@cbourgois could you please resolve your conflits?

I encourage you to add a `.npmrc` file:
```sh
$ cat .npmrc
package-lock=false
```

**Comment by cbourgois** [comment]
@antleblanc `.npmrc` added.

**Comment by antleblanc** [comment]
There is no need to release `/services` 2api until now. So I will merge and create a new release.

**Pull request merged**